### PR TITLE
Moves autowrap tests in with MATPLOTLIB/THEANO/ASCII tests.

### PR DESCRIPTION
**Pull request by moorepants**
- Uninstall cycle is now under the SAGE tests.
- autowrap related tests run with matplotlib, theano, an ascii tests.
- TEST_AUTOWRAP now uses conda installed packages.

<If this pull request fixes an issue please indicate which issue by typing "Fixes #NNNN" below.>

### CODE REVIEW

**Comment by moorepants** [comment]
This is ready to be merged. Can I get a quick review from someone?

File: .travis.yml [hunk 1]
```diff
@@ -306,16 +262,20 @@ before_install:
         conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION matplotlib numpy scipy pip llvmlite;
         source activate test-environment;
     fi
-
+  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
+        conda config --add channels conda-forge;
+        conda config --add channels symengine;
+        conda install python-symengine=0.2.0;
```
Comment (asmeurer) [inline .travis.yml]: Is it necessary to fix the version here?

Comment (asmeurer) [inline .travis.yml]: Oh I guess that was there before.

**Comment by asmeurer** [comment]
Thanks. This removes two unnecessary builds (just in time for us to add 5 more for Python 3.6).

**Pull request merged**